paper/src/main/java/com/denizenscript/denizen/paper/properties/EntityFriction.java

```diff
@@ -0,0 +1,88 @@
+package com.denizenscript.denizen.paper.properties;
+
+import com.denizenscript.denizen.objects.EntityTag;
+import com.denizenscript.denizencore.objects.core.ElementTag;
+import com.denizenscript.denizencore.objects.properties.Property;
+import com.denizenscript.denizencore.objects.ObjectTag;
+import com.denizenscript.denizencore.objects.properties.PropertyParser;
+import io.papermc.paper.entity.Frictional;
+import net.kyori.adventure.util.TriState;
+
+public class EntityFriction implements Property {
+
+    public static boolean describes(ObjectTag object) {
+        return object instanceof EntityTag entityTag && entityTag.getBukkitEntity() instanceof Frictional;
+    }
+
+    public static EntityFriction getFrom(ObjectTag entity) {
+        if (!describes(entity)) {
+            return null;
+        }
+        return new EntityFriction((EntityTag) entity);
+    }
+
+    private EntityFriction(EntityTag ent) {
+        entity = ent;
+    }
+
+    EntityTag entity;
+
+    public Boolean getFrictional() {
+        return ((Frictional) entity.getBukkitEntity()).getFrictionState().toBoolean();
+    }

```

Comment:
The method should return the `Frictional` interface, then it could be used in both the mech, tag, `getPropertyString`, etc.

Reply:
done